cdap-ui/cypress/integration/pipeline.studio.scheduling.spec.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+import { dataCy } from '../helpers';
+import {
+    loginIfRequired,
+    getArtifactsPoll,
+} from '../helpers';
+let headers = {};
+const PIPELINE_NAME = `schedule_test_pipeline_${Date.now()}`;
+const SCHEDULE_REPEAT_HOURLY = 'Hourly';
+const SCHEDULE_REPEAT_MONTHLY = 'Monthly';
+const SCHEULE_REPEAT_DAY_OF_MONTH = '5';
+const SCHEDULE_REPEAT_HOURS = "2";
+const SCHEDULE_MAX_CONCURRENT_RUNS = "3";
+describe('Pipeline Studio', () => {
```

Comment:
It would be helpful to be a little more specific and mention pipeline scheduling here, since we have multiple sets of tests for Pipeline Studio.